src/renderer/components/DownloadsDropdown/DownloadsDropdownButton.tsx

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) 2016-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import classNames from 'classnames';
+import React, {useEffect} from 'react';
+
+import {CLOSE_DOWNLOADS_DROPDOWN, CLOSE_DOWNLOADS_DROPDOWN_MENU, OPEN_DOWNLOADS_DROPDOWN} from 'common/communication';
+
+import '../../css/components/DownloadsDropdown/DownloadsDropdownButton.scss';
+
+type Props = {
+    isDownloadsDropdownOpen: boolean;
+    darkMode: boolean;
+    showDownloadsBadge: boolean;
+}
+
+const DownloadsDropDownButtonBadge = ({show}: { show: boolean }) => (
+    show ? <span className='DownloadsDropdownButton__badge'/> : null
+);
+
+const DownloadsDropdownButton: React.FC<Props> = ({darkMode, isDownloadsDropdownOpen, showDownloadsBadge}: Props) => {
+    const buttonRef: React.RefObject<HTMLButtonElement> = React.createRef();
+
+    useEffect(() => {
+        if (!isDownloadsDropdownOpen) {
+            buttonRef.current?.blur();
+        }
+    }, [isDownloadsDropdownOpen, buttonRef]);
+
+    const handleToggleButton = (event: React.MouseEvent<HTMLButtonElement>) => {
+        event.preventDefault();
+        event.stopPropagation();
+        if (isDownloadsDropdownOpen) {
+            window.ipcRenderer.send(CLOSE_DOWNLOADS_DROPDOWN);
```

Comment:
Nit: Keep the `ipcRenderer` calls in `MainPage` so that we only have one point where this is used. I'd rather not actually have `ipcRenderer` exposed and eventually I'll want to remove it so it would be easier if we didn't have references everywhere.